lib/index.js

```diff
@@ -4,7 +4,7 @@ const util        = require('./util');
 const sig         = require('./sig');
 const request     = require('miniget');
 const m3u8stream  = require('m3u8stream');
-const parseTime   = require('m3u8stream/lib/parse-time');
+const parseTime   = require('m3u8stream/dist/parse-time');
```

Comment:
+1 on the solution

Reply:
yep it's working with it! Merge it please ! :pray: